src/node-file-trace.js

```diff
@@ -27,7 +27,7 @@ module.exports = async function (files, opts = {}) {
   await Promise.all(files.map(file => {
     const path = resolve(file);
     job.emitFile(job.realpath(path), 'initial');
-    if (path.endsWith('.js') || path.endsWith('.node') || job.ts && path.endsWith('.ts'))
+    if (path.endsWith('.js') || path.endsWith('.node') || job.ts && (path.endsWith('.ts') || path.endsWith('.tsx')))
```

Comment:
Thanks for finding and fixing this issue! Can you add a unit test for this?

Reply:
Sure! Give me a bit to figure out how your tests are organized and I'll push something up.